homeassistant/components/zwave/climate.py

```diff
@@ -289,3 +289,11 @@ def set_swing_mode(self, swing_mode):
         if self._zxt_120 == 1:
             if self.values.zxt_120_swing_mode:
                 self.values.zxt_120_swing_mode.data = swing_mode
+
+    @property
+    def device_state_attributes(self):
+        """Return the optional state attributes."""
+        data = super().device_state_attributes
+        if self.fan_state:
+            data[ATTR_FAN_STATE] = self.fan_state
```

Comment:
call this `fan_action`, could be that we add this later like the `hvac_actions`

Reply:
I changed all references from fan "state" to "action" and I was able to see the new attribute in my install so I think everything should be working. I can squash all the commits down to one unless there are more changes?